Documentation/security/policy/language.rst

```diff
@@ -174,6 +174,29 @@ Note that while the above examples allow all egress traffic from an endpoint, th
 of the egress traffic may have ingress rules that deny the traffic. In other words,
 policy must be configured on both sides (sender and receiver).
 
+Simple Egress Deny
+~~~~~~~~~~~~~~~~~~
+
+The following example illustrates how to deny communication to endpoints with
+the label ``role=backend`` from endpoints with the label ``role=frontend``.
+If an ``egressDeny`` rule matches, egress traffic is denied even if the policy
+contains ``egress`` rules that would otherwise allow it.
+
+.. only:: html
+
+   .. tabs::
+     .. group-tab:: k8s YAML
+
+        .. literalinclude:: ../../../examples/policies/l3/egress-deny/egress-deny.yaml
+          :language: yaml
+     .. group-tab:: JSON
+
+        .. literalinclude:: ../../../examples/policies/l3/egress-deny/egress-deny.json
+
+.. only:: epub or latex
+
+        .. literalinclude:: ../../../examples/policies/l3/egress-deny/egress-deny.json
```

Comment:
https://github.com/cilium/cilium/pull/40212 just got merge (see commit https://github.com/cilium/cilium/commit/958bcab39a62e6a56284da63e54bd649a72ed225). So please align this example with the new format for the others:

```rst
.. literalinclude:: .. literalinclude:: ../../../examples/policies/l3/simple/l3.yaml
  :language: yaml
```